students/arun_nalla/session08/circle.py

```diff
@@ -0,0 +1,120 @@
+#! usr/bin/env phtyon
+
+""" Circle class assignment by
+Arun Nalla 12/08/2018"""
+
+from math import pi
+import random
+
+class Circle():
+
+    def __init__(self, radius):
+        if radius >= 1:
+            self.radius = radius
+        else:
+            raise ValueError ('Value is less than or equal to zero')
+
+    @property
+    def diameter(self):
+        self.diameter = self.radius * 2
+        return self.radius
```

Comment:
Why return `self.radius`, after computing `self.diameter`? Also you don't need to save the computed result to an attribute (`self.diameter`). You can just return the computed value as is (`return self.radius * 2`).